test/client/integration.test.ts

```diff
@@ -26,6 +27,14 @@ import * as mcl from "../../ts/mcl";
  * and reach a correct end state.
  */
 describe("Client Integration", function() {
+    before(async () => {
+        await del("./leveldb");
+    });
+
+    after(async () => {
+        await del("./leveldb");
+    });
```

Comment:
```suggestion
    before(async function () {
        await del("./leveldb");
    });

    after(async function () {
        await del("./leveldb");
    });
```
See https://mochajs.org/#arrow-functions . Generally in our test suite it will not matter, but good to follow Mocha's recommendations. Arrow funcs are also generally fine as callbacks for `descibe` blocks.

Reply:
ah, wasn't aware of this. will make changes.